mediathread/djangosherd/models.py

```diff
@@ -327,8 +327,11 @@ class SherdNote(Annotation):
     objects = SherdNoteManager()
 
     title = models.CharField(blank=True, max_length=1024, null=True)
-    asset = models.ForeignKey(Asset, related_name="sherdnote_set")
-    author = models.ForeignKey(User, null=True, blank=True)
+    asset = models.ForeignKey(
+        Asset, related_name="sherdnote_set",
+        on_delete=models.CASCADE)
+    author = models.ForeignKey(
```

Comment:
Wondering about PROTECT here? If the author is deleted, I think the annotations should cascade.

Reply:
Yeah - this has been updated.